feature-group-definitions/picture-in-picture.yml

```
@@ -1,5 +1,5 @@
-name: Picture-in-picture
-description: The picture-in-picture API allow websites to create a floating, always-on-top video window.
+name: Video picture-in-picture
```

Suggesting this instead, though I don't think we've done anything like this except for initial support:

```suggestion
name: Picture-in-picture (video)
```

My thinking here is that:

- People already call this "picture-in-picture" (and it directly analogizes to non-web TV technologies) but I couldn't find any good examples of this being called "video picture-in-picture". We probably shouldn't invent a backformation ourselves.
- I suspect "document picture in picture" will be a marked form for some time. Variants like "[Document in Picture-in-Picture](https://developer.chrome.com/docs/web-platform/document-picture-in-picture)" imply that document is stuffing something extra or unusual into the PiP window.

---

The parenthetical works I think, that's enough to make it clear if the two PiP features are seen by name only.